### PR TITLE
RTL8720E peripherals implementation update 

### PR DESCRIPTION
**Pull request by fengyunlai**

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@fengyunlai There are duplicated commits,
```
build/: 1st commit for build, configs/scripts/prebuild binaries for a…
os: 2nd commit for bsp. 
os/: 3rd commit for libs mk and docker related files
os/: 4th commit for adding libs in ProtectedLibs.mk, and add binary n… 
board/rtl8720e/src/libs/: 5th commit to add neccessary libraries
```
Could you rebase your private repo?

**Comment by sunghan-chang** [comment]
@fengyunlai New commit - https://github.com/Samsung/TizenRT/pull/5559/commits/bca879ff700194444a5c7519bf73ad3bf40a3044 - has file permission changed to 755. Could you keep them with 644?

**Comment by sunghan-chang** [comment]
And I think they are Windows format (Entire lines show changed). Could you check this as well?

**Comment by fengyunlai** [comment]
@sunghan-chang Sorry for my mistakes, now I modified in linux and also did the rebase properly.

File: os/board/rtl8720e/src/component/mbed/targets/hal/rtl8720e/flash_api.c [hunk 1]
```diff
@@ -47,9 +47,9 @@ void flash_write_protect(flash_t *obj, u32 protect)
 	/* To avoid gcc warnings */
 	(void) obj;
 
-	FLASH_Write_Lock();
+	//FLASH_Write_Lock();
```
Comment (sunghan-chang) [inline os/board/rtl8720e/src/component/mbed/targets/hal/rtl8720e/flash_api.c:50]: Could you let me know why they are commented out?

**Review by sunghan-chang — APPROVED** [review]

File: os/board/rtl8720e/src/component/mbed/targets/hal/rtl8720e/flash_api.c (hunk 1 above)
Comment (sunghan-chang) [reply os/board/rtl8720e/src/component/mbed/targets/hal/rtl8720e/flash_api.c:50]: I found your description.
```
Fixed UART & disabled flash lock/unlock in api

flash lock/unlock at this stage will hang the system at this stage
```

**Pull request merged**